frontend/src/main/components/Commons/Profits.js

```diff
@@ -14,7 +14,7 @@ const dummyData = [
 const Profits = () => {
     return (
         <Card>
-            <Card.Header as="h5">Profits</Card.Header>
+            <Card.Header as="h5"><strong>Profits</strong></Card.Header>
```

Comment:
The styling class could be used here as well (that way, you can edit the styling in one place, and it will be reflected everywhere).